templates/base/package.json

```diff
@@ -14,10 +14,11 @@
     "storybook:install": "cd storybook && yarn",
     "storybook:run": "cd storybook && yarn storybook",
     "storybook:build": "cd storybook && yarn build-storybook",
-    "generate": "plop --plopfile generators/index.js",
+    "generate": "plop --plopfile generators/index.js && npm run pretty-quick",
     "bundlesize": "bundlesize --config ./config/bundlesize.json",
     "analyze": "cross-env ANALYZE=true next build",
-    "perf": "docker-compose -f ./config/performance/sitespeed-compose.yml run sitespeed.io  --config sitespeed.config.json --budget.configPath sitespeed.budget.config.json --graphite.host=graphite urls.txt"
+    "perf": "docker-compose -f ./config/performance/sitespeed-compose.yml run sitespeed.io  --config sitespeed.config.json --budget.configPath sitespeed.budget.config.json --graphite.host=graphite urls.txt",
+    "pretty-quick": "pretty-quick"
```

Comment:
I don't see pretty-quick or prettier installed in the dependencies.

Reply:
Added the dependencies